.rubocop.yml

```diff
@@ -82,6 +86,12 @@ Metrics/ParameterLists:
   Max: 4
 Metrics/PerceivedComplexity:
   Max: 8
+Naming/FileName:
+  Enabled: false
```

Comment:
What fails for this one?

Reply:
From their docs,
> This cop makes sure that Ruby source files have snake_case names. Ruby scripts (i.e. source files with a shebang in the first line) are ignored.

and flags us for the following:
    - 'Gemfile'
    - 'Rakefile'
    - 'lib/theme_template/Gemfile'
    - 'test/fixtures/test-dependency-theme/test-dependency-theme.gemspec'
    - 'test/fixtures/test-theme/test-theme.gemspec'

which IMO is a bug because this cop existed as an `enabled-by-default-cop` earlier known as `Style/FileName`

Reply:
Yeah... that looks like a bug. Weird.